### DISCUSSION TITLE
Institutional email verification

### DISCUSSION THREAD

**Discussion by Risharpls** [RosettaCommons/rosetta#94] [ROSIE] [answered]
Hello,
I'd like to access ROSIE-2 but I'm getting an email verification error. I am currently employed in the Department of Biotechnology at the University of Antofagasta (www.uantof.cl). I would like to kindly request access to your platform to facilitate the continuation of my research Thesis.
Thank you in advance.
Kind regards.

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know, - this should be fixed now! Please feel free to re-open if you still have trouble logging in.

**Reply by Risharpls** [reply]
Thank you!!
